requirements.txt

```diff
@@ -1,21 +1,61 @@
+simplejson==3.17.0 \
+    --hash=sha256:0fe3994207485efb63d8f10a833ff31236ed27e3b23dadd0bf51c9900313f8f2 \
+    --hash=sha256:17163e643dbf125bb552de17c826b0161c68c970335d270e174363d19e7ea882 \
+    --hash=sha256:1d1e929cdd15151f3c0b2efe953b3281b2fd5ad5f234f77aca725f28486466f6 \
+    --hash=sha256:1d346c2c1d7dd79c118f0cc7ec5a1c4127e0c8ffc83e7b13fc5709ff78c9bb84 \
+    --hash=sha256:1ea59f570b9d4916ae5540a9181f9c978e16863383738b69a70363bc5e63c4cb \
+    --hash=sha256:1fbba86098bbfc1f85c5b69dc9a6d009055104354e0d9880bb00b692e30e0078 \
+    --hash=sha256:229edb079d5dd81bf12da952d4d825bd68d1241381b37d3acf961b384c9934de \
+    --hash=sha256:22a7acb81968a7c64eba7526af2cf566e7e2ded1cb5c83f0906b17ff1540f866 \
+    --hash=sha256:2b4b2b738b3b99819a17feaf118265d0753d5536049ea570b3c43b51c4701e81 \
+    --hash=sha256:4cf91aab51b02b3327c9d51897960c554f00891f9b31abd8a2f50fd4a0071ce8 \
+    --hash=sha256:4fd5f79590694ebff8dc980708e1c182d41ce1fda599a12189f0ca96bf41ad70 \
+    --hash=sha256:5cfd495527f8b85ce21db806567de52d98f5078a8e9427b18e251c68bd573a26 \
+    --hash=sha256:60aad424e47c5803276e332b2a861ed7a0d46560e8af53790c4c4fb3420c26c2 \
+    --hash=sha256:7739940d68b200877a15a5ff5149e1599737d6dd55e302625650629350466418 \
+    --hash=sha256:7cce4bac7e0d66f3a080b80212c2238e063211fe327f98d764c6acbc214497fc \
+    --hash=sha256:8027bd5f1e633eb61b8239994e6fc3aba0346e76294beac22a892eb8faa92ba1 \
+    --hash=sha256:86afc5b5cbd42d706efd33f280fec7bd7e2772ef54e3f34cf6b30777cd19a614 \
+    --hash=sha256:87d349517b572964350cc1adc5a31b493bbcee284505e81637d0174b2758ba17 \
+    --hash=sha256:8de378d589eccbc75941e480b4d5b4db66f22e4232f87543b136b1f093fff342 \
+    --hash=sha256:926bcbef9eb60e798eabda9cd0bbcb0fca70d2779aa0aa56845749d973eb7ad5 \
+    --hash=sha256:9a126c3a91df5b1403e965ba63b304a50b53d8efc908a8c71545ed72535374a3 \
+    --hash=sha256:ad8dd3454d0c65c0f92945ac86f7b9efb67fa2040ba1b0189540e984df904378 \
+    --hash=sha256:d140e9376e7f73c1f9e0a8e3836caf5eec57bbafd99259d56979da05a6356388 \
+    --hash=sha256:da00675e5e483ead345429d4f1374ab8b949fba4429d60e71ee9d030ced64037 \
+    --hash=sha256:daaf4d11db982791be74b23ff4729af2c7da79316de0bebf880fa2d60bcc8c5a \
+    --hash=sha256:f4b64a1031acf33e281fd9052336d6dad4d35eee3404c95431c8c6bc7a9c0588 \
+    --hash=sha256:fc046afda0ed8f5295212068266c92991ab1f4a50c6a7144b69364bdee4a0159 \
+    --hash=sha256:fc9051d249dd5512e541f20330a74592f7a65b2d62e18122ca89bf71f94db748
+statsd==3.3.0 \
+    --hash=sha256:c610fb80347fca0ef62666d241bce64184bd7cc1efe582f9690e045c25535eaa \
+    --hash=sha256:e3e6db4c246f7c59003e51c9720a51a7f39a396541cb9b147ff4b14d15b5dd1f
+swagger-parser==1.0.1 \
+    --hash=sha256:0802ea49468fab102ed8cb8a6cc6c41423ad60fd396c14a2f187c7d5165f2e18 \
+    --hash=sha256:8c0263e3128d3af9150c6fcd83d768967211f3bcb4da330fbe318fb44e38bdd2
+uWSGI==2.0.18 \
+    --hash=sha256:4972ac538800fb2d421027f49b4a1869b66048839507ccf0aa2fda792d99f583
+WSGIProxy2==0.4.6 \
```

Comment:
``wsgiproxy`` appears to only be used in ``mozsvc/tests/support.py``, maybe it can move to ``requirements-test.txt`` as well?